examples/with-yoga/package.json

```
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@gympass/yoga": "^7.28.0",
+    "next": "latest",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "styled-components": "^4.4.1"
+  },
+  "devDependencies": {
+    "eslint": "latest",
+    "eslint-config-next": "latest"
+  }
```

```suggestion
  }
```
Let's leave the `eslint` dependencies out by default similar to [other examples](https://github.com/vercel/next.js/blob/canary/examples/blog-starter/package.json).